gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -229,6 +230,9 @@ public Boolean load(String bucketName) {
   // GCS access instance.
   @VisibleForTesting Storage storage;
```

Comment:
Do we need `storage` when we have `storageRequestFactory` already?

Reply:
`Storage` is still required for creating other request in `Objects` or `Buckets`. Ideally we can have the factory to create all the requests so that we have a unified interface to deal with them.

Reply:
I think we can expose `storage` from `storageRequestFactory` via getter instead?

Reply:
Good idea, and it looks like `Storage` instance is only used for creating storage requests. I'll make the change. We can migrate the creation of other requests into the factory latter if we choose to do so, but this will clean it up a bit for now. (Migrating all request creation into it feels like deserve its own PR to me).

Reply:
It doesn't make sense to hide Storage now since some classes (GoogleCloudStorageReadChannel, for example) still requires a Storage to construct, and it doesn't make sense to hide it in Impl class, get it out, construct a read channel and hide it again in read channel. It might make relation between those objects a little bit strange.